experiment/simulation/pages/factorial.html

```diff
@@ -0,0 +1,242 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Recursion | VLabs</title>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/virtual-labs/virtual-style@0.0.8-b/css/style.min.css">
+    <link rel="stylesheet" type="text/css" href="../css/main.css" />
+</head>
+
+<body>
+    <div class="mdl-layout mdl-js-layout mdl-layout--fixed-drawer">
+        <div class="mdl-layout__drawer">
+            <span class="mdl-layout-title"><a href="../index.html" style="color:black;">Problems</a></span>
+            <nav class="mdl-navigation">
+                <a class="mdl-navigation__link" href="./add_numbers.html">Sum of Numbers</a>
+                <a class="mdl-navigation__link" href="./power.html">Power of a Number</a>
+                <!-- <a class="mdl-navigation__link" href="./towerofhanoi.html">Visualize Towers of Hanoi</a> -->
+                <a class="mdl-navigation__link nav-active" href="./factorial.html">Factorial of Number</a>
+                <a class="mdl-navigation__link" href="./fibonacci.html">Fibonacci series</a>
+            </nav>
+        </div>
+        <main class="mdl-layout__content">
+            <div class="demo-separator"></div>
+            <div class="mdl-grid">
+                <div class="mdl-cell mdl-cell--8-col mdl-cell--8-col-tablet mdl-cell--4-col-phone">
+                    <div class="v-collapsible-instruction-container v-instruction-container">
+                        <div class="v-collapsible is-size-5" id="task-description">
+                            Instructions
+                        </div>
+                        <div class="v-content" style="text-align: left; padding-left: 10%; padding-right: 10%;">
+                            <ul style="list-style: disc;">
+                                <li>Select the most appropriate option from the dropdowns in the code.</li>
```

Comment:
Try to keep configuration separate from code for better maintainability. If instructions need to be changed, the code shouldn't need to be touched.

Reply:
I took the same configuration as used in other experiments of different labs. I will try to add it as a separate file, if possible.